### DISCUSSION TITLE
X11 Forwarding?

### DISCUSSION THREAD

**Discussion by CesarPetrescu** [Eugeny/tabby#4183] [Q&A] [answered]
Is it implemented yet? Or are there any plugins for it? [Found out anyways]

**Comment by CesarPetrescu** [comment, marked as answer]
Nevermind,you can enable x11 forwarding under settings>profiles>[ssh connection]>advanced>x11 forwarding

**Comment by lawweiliang** [comment]
@CesarPetrescu   
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
